### PR TITLE
[2.0.12] Using ng-model with search-box

### PR DESCRIPTION
**Pull request by kandizzy**
#1076

ngModel can be passed to the search-box directive. See `example/search-box-ngmodel.html` for example implementation.

### CODE REVIEW

File: src/coffee/directives/search-box.coffee [hunk 1]
```diff
@@ -19,6 +19,7 @@ angular.module('uiGmapgoogle-maps')
 'uiGmapSearchBoxParentModel', '$http', '$templateCache', '$compile',
   (GoogleMapApi, Logger, SearchBoxParentModel, $http, $templateCache, $compile) ->
     class SearchBox
+      require: '^ngModel'
```
Comment (cthrax) [inline src/coffee/directives/search-box.coffee]: Shouldn't this be "ngModel"? Since it's a sibling of uiGmapSearchBox? Rather than a parent?

Comment (kandizzy) [inline src/coffee/directives/search-box.coffee]: Let me try it without the `^` to be sure

Comment (kandizzy) [inline src/coffee/directives/search-box.coffee]: yes "ngModel" works. just pushed an update

Comment (cthrax) [inline src/coffee/directives/search-box.coffee]: Cool, nice work on this!

Comment (kandizzy) [inline src/coffee/directives/search-box.coffee]: Thanks! :)

File: website_libs/dev_deps.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*!
- * Bootstrap without jQuery v0.5.0 for Bootstrap 3
+ * Bootstrap without jQuery v0.4.1 for Bootstrap 3
```
Comment (nmccready) [inline website_libs/dev_deps.js]: was this supposed to change again?

Comment (nmccready) [inline website_libs/dev_deps.js]: I need to add this to curl.json so this never happens.

**Comment by nmccready** [comment]
LGTM besides the jquery bootstrap.

File: website_libs/dev_deps.js (hunk 2 above)
Comment (kandizzy) [inline website_libs/dev_deps.js]: Oh yeah, I thought I fixed this up. Just pushed the correct version.

**Pull request merged**